include/Core/DB/TableManager.php

```diff
@@ -484,6 +500,7 @@ private function get_create_table_query() {
 				attachment_name VARCHAR(1000),
 				ip_address VARCHAR(15),
 				result TINYINT(1),
+				error_message TEXT,
```

Comment:
@sudar,

I've used `TEXT` since that looks suitable.

![image](https://user-images.githubusercontent.com/17510033/62263748-fbf8cc00-b43a-11e9-82ee-539d1daa0a3f.png)

Reply:
The major difference between `Text` and `varchar` is that, `varchar` is best suited for lookups when compared with `Text`.

What are the typical values that we will be storing in these columns?

Reply:
@sudar,

We shall be storing the PHPMail exception messages which extends from Exception class

https://www.php.net/manual/en/exception.getmessage.php

If `VARCHAR` is best suited, then I guess we could use `VARCHAR(1000)` for the `error_message` column.

Reply:
@mariadanieldeepak 

Yeah, let's use `varchar(1000)`